app/controllers/attachments_controller.rb

```diff
@@ -2,5 +2,9 @@ class AttachmentsController < ApplicationController
   def index
     @edition = Edition.find_current(document: params[:document])
     assert_edition_state(@edition, &:editable?)
+
```

Comment:
Minor: the commit message is slightly different to the code, in that the changes don't mention pre-release.